src/components/items/item.js

```diff
@@ -29,7 +29,7 @@ const Item = ({
       color="primary"
       checked={checked}
       indeterminate={indeterminate}
-      disabled={disabled}
+      disabled={disabled || item.disabled === true}
```

Comment:
@MiguelTolosa sorry last few comments I hope!
This line seems like a duplication of the same logic at list.js@74
You're going to pass a disabled=true from the list so I'm not sure there's a reason to check it again.

Reply:
yup, that's the last comment :)